src/guides/v2.4/install-gde/system-requirements-tech.md

```diff
@@ -33,7 +33,7 @@ Upgrading the Magento applications and extensions you obtain from Magento Market
 
 ## Database
 
-MySQL 5.6, 5.7
+MySQL 5.7
```

Comment:
If the minimum is 5.7.9 we should say so here.

Reply:
5.7.9 is a minimal stable version of 5.7 and magento supports only stable versions of a software. Anyway, it's better to clarify! Fixed.